tests/integration.rs

```diff
@@ -617,6 +618,20 @@ fn test_handle_and_sign_ping_pong() {
     });
 }
 
+#[test]
+fn test_buffer_underflow_sign_proposal() {
+    let key_type = KeyType::Consensus;
+    ProtocolTester::apply(&key_type, |mut pt| {
+        send_buffer_underflow_request(&mut pt);
+        let response: Result<(), ()> = match read_response(&mut pt) {
+            proto::privval::message::Sum::SignedProposalResponse(_) => Ok(()),
+            other => panic!("unexpected message type in response: {other:?}"),
+        };
+
+        assert!(response.is_ok());
+    });
+}
```

Comment:
Seems this test is failing in CI. If we can get it green I can merge this.

Reply:
Hi @tony-iqlusion I fixed the typo in filenames, now all the tests are green

Reply:
Thanks. Headed out for the rest of the week, but I will try to get this merged next week.

Reply:
I feel today is great day to get this merged :)